core/src/main/java/org/mule/endpoint/MuleEndpointURI.java

```diff
@@ -494,37 +498,70 @@ public String getFilterAddress()
 
     public String getUser()
```

Comment:
Add javadoc to the implemented method in the interface to indicate that the returned value is url-decoded.
Also add @Override